frontend/e2e/tests/versioning-tests.ts

```diff
@@ -31,8 +31,9 @@ export default async () => {
     await waitForElementVisible(byId('features-page'))
     await click('#env-settings-link')
     await click(byId('enable-versioning'))
-    await click('#confirm-toggle-feature-btn')
-    await waitAndRefresh(10000)
+    await click('#confirm-btn-yes')
+    await waitAndRefresh(1000)
+    await refreshUntilElementVisible(byId('feature-versioning-enabled'))
```

Comment:
Do we need both of these still?

Reply:
Actually yeah since we have the FE change we can wait without refreshing at all. I'll keep the refreshUntilElementVisible util since I think we can use it in future.